lib/docker-delta.coffee

```diff
@@ -120,11 +132,13 @@ exports.applyDelta = (srcImage, dstImage) ->
 		Promise.attempt ->
 			switch dockerDriver
 				when 'btrfs'
-					btrfs.deleteSubvolAsync(dstRoot)
-					.then ->
-						btrfs.snapshotSubvolAsync(srcRoot, dstRoot)
+					if srcRoot?
```

Comment:
Hm, but I still want to throw if it's not a supported driver. I think it's ok as is